### PR TITLE
[composer.json] phpunit upgrade and replace deprecated methods

### PR DESCRIPTION
**Pull request by maltheism**
## Brief summary of changes

phpunit upgrade and replace deprecated methods

### CODE REVIEW

File: composer.json [hunk 1]
```diff
@@ -6,21 +6,21 @@
         "smarty/smarty" : "~3.1.33",
         "firebase/php-jwt" : "~3.0",
         "google/recaptcha": "~1.1",
-        "php-http/guzzle6-adapter": ">=2.0.0",
+        "php-http/guzzle7-adapter": "0.1.1",
         "psr/http-message": "~1.0",
         "psr/http-server-handler" : "*",
         "psr/http-server-middleware" : "*",
-        "laminas/laminas-diactoros" : "~1.8.4",
+        "laminas/laminas-diactoros" : "2.5.0",
```
Comment (driusan) [inline composer.json:13]: Do all these other libraries need to change to update phpunit?

**Comment by driusan** [comment]
According to GitHub actions this seems to be missing the changes from maltheism/Loris#9 and the changes to the setup/teardown signatures in #7157

**Comment by driusan** [comment]
This has conflicts in composer.lock because #7166 was merged.

File: composer.json (hunk 1 above)
Comment (kongtiaowang) [reply composer.json:13]: No, there is no need to update other libraries.

**Comment by kongtiaowang** [comment]
As @driusan said, @maltheism you have a lot of work to update php 8.0. This PR can make it easier for you.
https://github.com/aces/Loris/pull/7170

**Comment by driusan** [comment]
Replaced by #7170

**Pull request closed without merging**